### PR TITLE
created cart page

### PR DESCRIPTION
**Pull request by agrawalshanu708**
add two section
1. show selected products
2. another for price details and place order
Screenshot (90)

### CODE REVIEW

File: Cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    display: flex;
+    flex-wrap: wrap;
+    
+  }
+  .final-card-container{
+    border: 1px solid black;
+  }
+
+.cart-container >div{
+    border: 1px solid #e0f2fe;
```
Comment (yashpatel1797) [inline Cart/cart.css:23]: use variable here.

**Review by therajatg — APPROVED** [review]
looks great

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge

File: Cart/cart.css (hunk 1 above)
Comment (agrawalshanu708) [reply Cart/cart.css:23]: OHKAY

**Pull request merged**